src/librustc_apfloat/ieee.rs

```diff
@@ -388,52 +388,54 @@ impl<S: Semantics> fmt::Display for IeeeFloat<S> {
         let _: Loss = sig::shift_right(&mut sig, &mut exp, trailing_zeros as usize);
```

Comment:
cc @eddyb

Reply:
Thanks for the ping!

@JohnTitor The indentation change is unfortunate, and I'm not sure using `Ordering` makes this more readable. But also, this code was ported from C++, and I'd rather not diverge too much (main exception being abstractions that avoid code duplication etc.).

Reply:
@eddyb Okay, I removed the change except for mentioned below.